ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -0,0 +1,306 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+public class CommandBar: UIView {
+    // Hierarchy:
+    //
+    // closeButton
+    // containerView
+    // |--layer.mask -> containerMaskLayer (fill containerView)
+    // |--subviews
+    // |  |--scrollView (fill containerView)
+    // |  |  |--subviews
+    // |  |  |  |--stackView
+    // |  |  |  |  |--buttons (fill scrollView content)
+
+    // MARK: Public Properties
+
+    public weak var delegate: CommandBarDelegate?
+
+    // MARK: Private Properties
+
+    private let barAppearance: CommandBarAppearance
+    private let itemGroups: [CommandBarItemGroup]
+    private let leadingItem: CommandBarItem?
```

Comment:
what's the importance of leading item and trailing item, leading button and trailing button to be stored separately?

Reply:
Removed.